modules/installation-installing-bare-metal.adoc

```diff
@@ -36,12 +36,12 @@ endif::restricted[]
 . Monitor the bootstrap process:
 +
 ----
-$ ./openshift-install --dir=<installation_directory> wait-for bootstrap-complete \ <1>
-    --log-level=info <2>
-
-INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
-INFO API v1.14.6+c4799753c up
-INFO Waiting up to 30m0s for the bootstrap-complete event...
+$ ./openshift-install --dir=/root/manual_installation/installation_directory wait-for bootstrap-complete \
```

Comment:
Should this stay `<installation_directory>` instead of `/root/manual_installation/`?

You removed the callouts here but not below the code block.

Reply:
Correct, changed to <installation_directory> and returned the callouts.